packages/components/Callout/macos/FRNCalloutManager.m

```diff
@@ -36,6 +38,28 @@ + (NSRect)screenRect:(id)json
 
 @interface RCT_EXTERN_MODULE(FRNCalloutManager, RCTViewManager)
 
+RCT_EXPORT_METHOD(focusWindow : (nonnull NSNumber *)viewTag)
+{
+	dispatch_async(dispatch_get_main_queue(), ^{
+		NSView *view = [self.bridge.uiManager viewForReactTag:viewTag];
+
+		if ([view isKindOfClass:[FRNCalloutView class]]) {
+			[(FRNCalloutView *)view focusWindow];
+		}
+	});
+}
+
+RCT_EXPORT_METHOD(blurWindow : (nonnull NSNumber *)viewTag)
+{
+	dispatch_async(dispatch_get_main_queue(), ^{
+		NSView *view = [self.bridge.uiManager viewForReactTag:viewTag];
+
+		if ([view isKindOfClass:[FRNCalloutView class]]) {
+			[(FRNCalloutView *)view blurWindow];
+		}
+	});
+}
```

Comment:
Instead of using the old Architecture's UIManager to add a UI thread task, let's just invoke it through `dispatch_async` directly. Also changed the type of view to `NSView` because I updated the return type in RNM 0.74.

```suggestion
RCT_EXPORT_METHOD(focusWindow : (nonnull NSNumber *)viewTag)
{
	dispatch_async(dispatch_get_main_queue(), ^{
		NSView *view = [self.bridge.uiManager viewForReactTag:viewTag];

		if ([view isKindOfClass:[FRNCalloutView class]]) {
			[(FRNCalloutView *)view focusWindow];
		}
	});
}

RCT_EXPORT_METHOD(blurWindow : (nonnull NSNumber *)viewTag)
{
	dispatch_async(dispatch_get_main_queue(), ^{
		NSView *view = [self.bridge.uiManager viewForReactTag:viewTag];

		if ([view isKindOfClass:[FRNCalloutView class]]) {
			[(FRNCalloutView *)view blurWindow];
		}
	});
}
```

Reply:
JK you did.

Reply:
done :)